rpcclient/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/decred/dcrd/dcrjson v1.0.0
 	github.com/decred/dcrd/dcrutil v1.1.1
 	github.com/decred/dcrd/gcs v1.0.1
-	github.com/decred/dcrd/wire v1.1.0
```

Comment:
This should not be bumped.  This does not introduce a new requirement on v1.2.0 of the wire package.